GOVERNANCE.md

```diff
@@ -0,0 +1,246 @@
+# Governance
+
+## Overview
+
+sktime is a meritocratic, consensus-based community project. Anyone with an interest in the project can join the community, contribute to the project, and participate in the governance process. This document describes how that participation takes place and how to set about earning merit within our community. 
+
+We are particularly motivated to support new and/or anxious collaborators, people who are looking to learn and develop their skills, and anyone who has experienced discrimination in the past.
+
+| Section | Purpose |
+|---|---|
+| [Code of Conduct](#Code-of-Conduct) | How we expect all members of the sktime community to interact |
+| [Roles](#Roles) | What roles we have in sktime's community and what rights and responsibilities they have | 
+| [Decision making](#Decision-making) | How we make decisions |
+| [Acknowledging contributions](#Acknowledging-contributions) | How we acknowledge contributions
+| [Outlook](#Outlook) | What we may change in the future |
+
+## Code of Conduct
+
+We value the participation of every member of our community and want to ensure an that every contributor has an enjoyable and fulfilling experience. Accordingly, everyone who participates in the sktime project is expected to show respect and courtesy to other community members at all times.
+
+To make clear what is expected, we ask all members of the community to conform to our [Code of Conduct](https://github.com/alan-turing-institute/sktime/blob/master/CODE_OF_CONDUCT.rst). 
+
+## Roles
+
+We distinguish between the following roles. For each role, we describe their rights and responsibilities, and appointment process in more detail below.
+
+|Role | Rights/responsibilities | Appointment |
+|---|---|---|
+| [Contributor](#Contributors) | - | concrete contribution |
+| [Algorithm maintainer](#Algorithm-maintainer) | algorithm maintenance, voting and veto right for changes to their algorithm | algorithm contribution or appointment by current maintainer |
+| [Core developer](#Core-developers) | direct write access, issue/PR management, veto right, voting, nomination | nomination by core developers, vote by core developers, 2/3 majority |
+| [CoC committee member](#CoC-committee-members) | CoC maintenance, investigation and enforcement | nomination by core developers, vote by core developers, 2/3 majority and simple CoC majority | 
+| [CC member](#CC-members) | conflict resolution, technical leadership, project management | nomination by core developers, vote by core developers, 2/3 majority and simple CC majority |
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the project. Anyone can become a contributor, and contributions can take many forms – not only code – as detailed in the [contributing guide](https://github.com/alan-turing-institute/sktime/blob/master/CONTRIBUTING.rst). 
```

Comment:
formulation may be important "contribute" or "have contributed"?
When does a contributor become a former contributor?